README.md

```diff
@@ -2,8 +2,147 @@
+
+Here is an example of the request body:
+```
+{
+	"batch": {
+		"id": "fghij67890",
+		"fileCount": 10
+	},
+	"callbackUrl": "https://transmission-listener.public.mdtp/upscan-listener/listen",
+	"requestTimeoutInSeconds": 300,
+	"file": {				
+		"reference": "abcde12345",
+		"checksum": "asdrfgvbhujk13579",
+		"location": "https://file-outbound-asderfvghyujk1357690.aws.amazon.com",
+		"name": "someFileN.ame",
+		"mimeType": "application/pdf",				
+		"sequenceNumber": 3
+	},
+	"journey":{
+		"name": "someJouney name",
+		"version": "1.0"
+	},
+	"properties":[
+		{
+			"name": "property1",
+			"value": "value1"
+		},
+		{
+			"name": "property2",
+			"value": "value2"
+		}
+	]			
+}
+```
+
+### Request outcomes
+
+If the POST is not successful, the service will return a HTTP error code (4xx, 5xx). The response body will contain XML encoded details of the problem. See the Error handling section for details.
+
+If the POST is successful, the service returns a HTTP 204 response with an empty body.
+
+**Please not that successful result means only that the request has been parsed and has been stored for further processing. Because MDG
+processing is performed asynchronously, consuming service should wait until callback from MDG is made to mark batch as processed successfully.***
+
+
+### Whitelisting client services
+
+Any service using `file-transmission` must be whitelisted. Please contact platform-services if you want to use the service.
+Consuming services must identify themselves in requests via the `User-Agent` header. If the supplied value is not in file-transmissions list of allowed services then the `/initiate` call will fail with a `403` error.
+
+In addition to returning a `403` error, Upscan will log details of the Forbidden request. For example:
+
+```json
+{
+    "app":"file-transmission",
+    "message":"Invalid User-Agent: [Some(my-unknown-service-name)].",
+    "logger":"application",
+    "level":"WARN"
+}
+```
+
+*Note:* If you are using `[http-verbs](https://github.com/hmrc/http-verbs)` to call Upscan, then the `User-Agent` header will be set automatically.
+(See: [HttpVerb.scala](https://github.com/hmrc/http-verbs/blob/2807dc65f64009bd7ce1f14b38b356e06dd23512/src/main/scala/uk/gov/hmrc/http/HttpVerb.scala#L53))
+
+
+# Running locally
+
+Service can be run locally by sbt (`sbt run`) or using service manager (`sm --start FILE_TRANSMISSION`)
+
+## Related projects, useful links:
+
+* [upscan](https://github.com/hmrc/upscan-initiate) - service that manages the process of uploading files to MDTP by end users
+
+### Testing
+* [file-transmission-acceptance-tests](https://github.com/hmrc/file-transmisson-acceptance-tests) - acceptance tests of the service
+
+### Slack
+* [#team-plat-services](https://hmrcdigital.slack.com/messages/C705QD804/)
+* [#event-upscan](https://hmrcdigital.slack.com/messages/C8XPL559N)
```

Comment:
Is the event-upscan relevant in this case, or would some sort of event-file-transmission channel need to be created?